Sources/Classes/core/MiniAppAnalytics.swift

```diff
@@ -208,6 +212,8 @@ public class MiniAppAnalytics {
             return .setCloseAlertInfo
         case .sendJsonToHostapp:
             return .sendJsonToHostapp
```

Comment:
You may need to add for `sendJsonToMiniApp` ?

Reply:
We need not add it as it is not the `MiniAppJSActionCommand` type. This method in analytics is to track the JSAction commands received form the JS-Bridge I guess.